### DISCUSSION TITLE
How to change visible action name in context menu to lowercase?

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#4513] [Q&A] [answered]
Hi all!

When I create an action "SomeAction" for example, it is shown as "Some Action" in a context menu. In German we need sometimes lowercase on the second or following words like "Some action" - is there a property or something else I can do in the action class to achieve this?

Many thanks!

**Comment by duncanmcclean** [comment, marked as answer]
You can use the `title` method to return whatever you like:

```php
public static function title()
{
    return 'WhateverYouWantTheTitleToBe';
}
```

**Reply by jensolafkoch** [reply]
Thanks, Duncan! :-)
